seaworthy/tests-pytest/test_pytest.py

```diff
@@ -0,0 +1,218 @@
+import docker
+import pytest
+
+from seaworthy.checks import docker_client
+from seaworthy.containers.base import ContainerBase
+from seaworthy.dockerhelper import DockerHelper, fetch_images
+from seaworthy.pytest.checks import dockertest
+from seaworthy.pytest.fixtures import (
+    clean_container_fixtures, container_fixture, docker_helper_fixture,
+    image_pull_fixture)
+
+
+IMG = 'nginx:alpine'
+
+
+def setup_module():
+    with docker_client() as client:
+        fetch_images(client, [IMG])
+
+
+# We redefine the docker_helper fixture here to have function scope so our
+# tests don't conflict.
+docker_helper = docker_helper_fixture(scope='function')
+
+
+@dockertest()
+class TestDockerHelperFixture:
+    def test_fixture(self, testdir):
+        """
+        The ``docker_helper`` fixture should automatically be in scope for a
+        project that uses this pytest plugin.  When the ``DockerHelper`` is
+        passed to a test function using a fixture, it should be possible to
+        create, start, stop, and remove a container using the helper.
+        """
+        testdir.makepyfile("""
```

Comment:
These tests are run in a separate process and so aren't seen in coverage reports.

It also means they have their own `docker_helper` fixture, that can conflict with the one in the process-local tests. Which means that
a) The local `docker_helper` fixture must have function scope
b) The local `docker_helper` fixture must not be used in these specific tests

Reply:
Maybe these should be in their own test module to keep them separate?